src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -860,11 +860,11 @@ trait BronzeTransforms extends SparkSessionWrapper {
   private[overwatch] def getAllEventLogPrefix(inputDataframe: DataFrame, apiEnv: ApiEnv): DataFrame = {
     try{
     val mountMap = getMountPointMapping(apiEnv) //Getting the mount info from api and cleaning the data
-      .withColumn("mount_point", when('mount_point.endsWith("/"), 'mount_point.substr(lit(0), length('mount_point) - 1)).otherwise('mount_point))
-      .withColumn("source", when('source.endsWith("/"), 'source.substr(lit(0), length('source) - 1)).otherwise('source))
+      .withColumn("mount_point", removeTrailingSlashes('mount_p))
```

Comment:
column `mount_p` doesn't exist this will break the code.

Reply:
yes you are 100% right. Mistake by my end while doing final push. By mistake the characters were removed my bad. But before doing the final push i have done a deployment and it worked fine.